guide/src/SUMMARY.md

```diff
@@ -37,47 +37,47 @@
     - [Standard Input and Output](running/stdio.md)
     - [Exceptions and Panics](running/panics.md)
 
-- [Samples](samples/overview.md)
-    - [first sample](samples/first/DESCRIPTION.md)
-    - [echo sample](samples/echo/DESCRIPTION.md)
-    - [fibonacci sample](samples/fibonacci/DESCRIPTION.md)
-    - [hello-world-include sample](samples/hello-world-include/DESCRIPTION.md)
-    - [hello-world-simple sample](samples/hello-world-simple/DESCRIPTION.md)
-    - [hello-world sample](samples/hello-world/DESCRIPTION.md)
-    - [line-echo sample](samples/line-echo/DESCRIPTION.md)
+- [Samples](../../samples/overview.md)
+    - [first sample](../../samples/first/DESCRIPTION.md)
+    - [echo sample](../../samples/echo/DESCRIPTION.md)
+    - [fibonacci sample](../../samples/fibonacci/DESCRIPTION.md)
+    - [hello-world-include sample](../../samples/hello-world-include/DESCRIPTION.md)
+    - [hello-world-simple sample](../../samples/hello-world-simple/DESCRIPTION.md)
+    - [hello-world sample](../../samples/hello-world/DESCRIPTION.md)
+    - [line-echo sample](../../samples/line-echo/DESCRIPTION.md)
```

Comment:
these should all work fine if you use the 'make guide' Makefile target.

The issue is that my travis job that builds the guide (using 'make guide') and then publishing it on GH Pages - is not working properly.

GH Pages  is just showing the MD files it finds in the folders, not the guide I build (html) using mdbook.